hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAUtils.java

```diff
@@ -146,7 +146,7 @@ public static String getSCMRatisDirectory(ConfigurationSource conf) {
 
   public static String getSCMRatisSnapshotDirectory(ConfigurationSource conf) {
     String snapshotDir =
-            conf.get(ScmConfigKeys.OZONE_SCM_HA_RATIS_STORAGE_DIR);
+            conf.get(ScmConfigKeys.OZONE_SCM_HA_RATIS_SNAPSHOT_DIR);
```

Comment:
Do this have any impact over upgrade? since snapshot directory path changed for user configured case.

Reply:
@sumitagrawl Thank you for the review.

I think this is a typo issue. A different place to store snapshot shouldn't cause difference.

Reply:
I mean,
Before upgrade, snapshot is stored at OZONE_SCM_HA_RATIS_STORAGE_DIR and snapshots are created in this path.
After upgrade, its changed to OZONE_SCM_HA_RATIS_SNAPSHOT_DIR.
In this case on startup, if ozone try to recover snapshots, will this cause any failure or impact to the system ?

Reply:
I think this snapshot directory acts like a tmp directory.  Every file downloaded from leader SCM to this directory, will be used to replace current SCM's rocksdb. So basically change it from  OZONE_SCM_HA_RATIS_STORAGE_DIR to OZONE_SCM_HA_RATIS_SNAPSHOT_DIR will not cause any compatibility issue, since the files left in this snapshot directory are not useful anymore after a service upgrade restart. 

@nandakumar131,  please correct me if I'm wrong.